### ISSUE TITLE
[a11y] The Elements on the program doesn't meet the minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by plabanTest1** [civiform/civiform#9604]
**Describe the bug**
The contrast between foreground and background colors doesn't meet the WCAG 2 AA minimum contrast ratio thresholds.

**To Reproduce**

1. Go to https://qa.civiform.dev/programs
2. Click on DevTools
3. Select Log in as: Civiform Admin
4. Click on the View button on any of the listed programs.
5. Run the Voiceover to the element and notice the readable text for it.

**Expected behavior**
Element has insufficient color contrast of 2.53 (foreground color: #9ca3af, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

**Screenshots**

![Image](https://github.com/user-attachments/assets/2c74a8fc-9a0e-400d-a11f-a5618d6ceeb1)

**Desktop (please complete the following information):**

- OS: [iOS]
- Browser [Chrome]
- Version [e.g. 131.0.6778.266]
- Accessibility Tools: [Parashu]

**Additional context**

**Comment by shreyachatterjee00** [comment]
Revisit after USWDS migration
